layouts/resources/baseof.html

```diff
@@ -63,6 +63,11 @@ <h1>
         {{end}}
         {{ partial "docs/inject/content-after" . }}
 
+        <!-- Nav links  -->
+        {{ if .IsPage }}
```

Comment:
lets add a comment here explaining what this does, and why we're limiting the nav links to content pages

the why is the tricky part